Frontend/src/Components/Shared/RangeRow.js

```diff
@@ -0,0 +1,65 @@
+import {Flex, Center, Text, Spacer, useColorMode} from "@chakra-ui/react";
+import getColor from "../Shared/colors";
+import RangeBar from "./RangeBar";
+
+export default function RangeRow(props) {
+    const { colorMode } = useColorMode();
+    const borderColor = getColor("border", colorMode);
+
+    const bg = (props.dataValue < props.dataConstant.MIN || props.dataValue > props.dataConstant.MAX) ?
+        getColor("errorBg", colorMode) : props.bg;
+
+    // set the decimal point to 3 by default
+    const decPoint = (props.DecimalPoint !== undefined) ? props.DecimalPoint : 3;
+    
+    let lineHeight;
+    let textSize;
+    let rangeBarHeight;
+    if (props.size === 'sm' || props.size === undefined) {
+        lineHeight = '1.2em';
+        textSize = 'sm';
+        rangeBarHeight = '1em';
+    } else if (props.size === 'md') {
+        lineHeight = '1.6em';
+        textSize = 'md';
+        rangeBarHeight = '1.4em';
+    } else if (props.size === 'lg') {
+        lineHeight = '2em';
+        textSize = 'lg';
+        rangeBarHeight = '1.8em';
+    }
```

Comment:
Let's add a default case for if `props.size` isn't one of the proper options (the default size should be 'sm')

Reply:
Line 18 should set `sm` as the default already—do you think I should be doing it another way?

Reply:
Never mind, just saw your later comment, I'll switch to using that syntax

Reply:
Sorry, I was proposing that there be a default case if a value was passed for `size`, but wasn't a valid value. However, that case should result in weird behavior, so you can ignore this thread.